test/functionalTest/cases/0876_attribute_dates/attrs_dates_overridden_by_user.test

```diff
@@ -34,12 +34,12 @@ brokerStart CB
 # NGSIv2 implementation notes). Maybe in the future could change.
 #
 # 01. Create entity with user provided dateCreated and dateModified metadata in a far away future
-# 02. GET /v2/entities/E1 and don't get dateModified or dateCreated metadata [should get user defined dates]
```

Comment:
Diff is pretty noisy in this file. I'd suggest to have a look to the final file and ensure it correct regarding what is told in section "User attributes or metadata matching builtin name" of ngsiv2_implementation_notes.md file.

Reply:
NTC